lib/pg.provider.js

```diff
@@ -153,7 +157,10 @@ class PostgresProvider extends StorageProvider {
         return;
       }
 
-      const error = this.gs.schema.validateCategory(category, obj);
+      const toValidate = Object.assign({}, obj);
+      delete toValidate.Id;
```

Comment:
I find this strange and inefficient, we should introduce some sort of mechanism to remove the need for this.
/cc @tshemsedinov @nechaido @belochub

Reply:
@lundibundi I will add a `TODO` comment here for now.